{"level":"error","message":"login failed login failed","stack":"Error: login failed\n    at module.exports.login (C:\\New folder\\Own-Project\\register-login API- Node.js project\\src\\routes\\auth\\controller.js:33:15)\n    at newFn (C:\\New folder\\Own-Project\\register-login API- Node.js project\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\New folder\\Own-Project\\register-login API- Node.js project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\New folder\\Own-Project\\register-login API- Node.js project\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at module.exports.validator (C:\\New folder\\Own-Project\\register-login API- Node.js project\\src\\routes\\controller.js:35:5)\n    at newFn (C:\\New folder\\Own-Project\\register-login API- Node.js project\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\New folder\\Own-Project\\register-login API- Node.js project\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\New folder\\Own-Project\\register-login API- Node.js project\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at middleware (C:\\New folder\\Own-Project\\register-login API- Node.js project\\node_modules\\express-validator\\lib\\middlewares\\check.js:16:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"thr is not defined thr is not defined","stack":"ReferenceError: thr is not defined\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:1)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"this.lazyrouter is not a function this.lazyrouter is not a function","stack":"TypeError: this.lazyrouter is not a function\n    at new use (C:\\New folder\\Own-Project\\register-login API- Node.js project\\node_modules\\express\\lib\\application.js:221:8)\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"e is not defined e is not defined","stack":"ReferenceError: e is not defined\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:1)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"Err is not defined Err is not defined","stack":"ReferenceError: Err is not defined\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:1)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"","stack":"Error\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"","stack":"Error\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"sth  sth ","stack":"Error: sth \n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"sth faild sth faild","stack":"Error: sth faild\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"sth faild sth faild","stack":"Error: sth faild\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"sth faild sth faild","stack":"Error: sth faild\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:26:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"sth faild sth faild","stack":"Error: sth faild\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"sth faild sth faild","stack":"Error: sth faild\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
{"level":"error","message":"sth faild sth faild","stack":"Error: sth faild\n    at Object.<anonymous> (C:\\New folder\\Own-Project\\register-login API- Node.js project\\index.js:31:7)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)\n    at node:internal/main/run_main_module:28:49"}
